app/modules/mdm_status/mdm_status_model.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use CFPropertyList\CFPropertyList;
+
+class Mdm_status_model extends \Model
+{
+    public function __construct($serial = '')
+    {
+        parent::__construct('id', 'mdm_status'); //primary key, tablename
+        $this->rs['id'] = '';
+        $this->rs['serial_number'] = $serial; //$this->rt['serial_number'] = 'VARCHAR(255) UNIQUE';
+        $this->rs['mdm_enrollment_type'] = '';
+        $this->rs['mdm_enrolled'] = '';
+        
+        // Add indexes
+        $this->idx[] = array('serial_number');
+        $this->idx[] = array('mdm_enrollment_type');
+        $this->idx[] = array('mdm_enrolled');
+
+        // Schema version, increment when creating a db migration
+        $this->schema_version = 1;
+        
+        // Create table if it does not exist
+       //$this->create_table();
+        
+        if ($serial) {
+            $this->retrieve_record($serial);
+        }
+        
+        $this->serial = $serial;
+    }
+    
+    
+    // ------------------------------------------------------------------------
+
+    /**
+     * Process data sent by postflight
+     *
+     * @param string data
+     *
+     **/
+    public function process($data)
+    {
+    	if (strpos($data, '<?xml') === false) {
+    		// old style txt file data has been passed - throw an error.
+    		throw new Exception("Error Processing Request: old format data found, please update the security module", 1);	
```

Comment:
no need for this

Reply:
Thanks, obviously leftovers from borrowed code.  I've removed it and updated the repo.